contracts/staking/PoSValidatorManager.sol

```diff
@@ -29,13 +29,11 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         /// @notice The reward calculator for this validator manager.
         IRewardCalculator _rewardCalculator;
         /// @notice Maps the validationID to a mapping of delegator address to delegator information.
-        mapping(bytes32 validationID => mapping(address delegator => Delegator)) _delegatorStakes;
+        mapping(bytes32 delegationID => Delegator) _delegatorStakes;
         /// @notice Maps the validationID to a mapping of delegator address to pending register delegator messages.
-        mapping(bytes32 validationID => mapping(address delegator => bytes))
-            _pendingRegisterDelegatorMessages;
+        mapping(bytes32 delegationID => bytes) _pendingRegisterDelegatorMessages;
         /// @notice Maps the validationID to a mapping of delegator address to pending end delegator messages.
-        mapping(bytes32 validationID => mapping(address delegator => bytes))
-            _pendingEndDelegatorMessages;
+        mapping(bytes32 delegationID => bytes) _pendingEndDelegatorMessages;
```

Comment:
Can you update the comments for these mappings?

Reply:
Done